Fix: Azure vault env variables (#6595)
Kong/docs.konghq.com: app/_src/gateway/kong-enterprise/secrets-management/backends/azure-key-vaults.md

}
```

> Note that Azure AD tenant ID, client ID, `vault_uri` and client secret need to be specified. 

Author: ```suggestion
The Azure AD tenant ID, client ID, `vault_uri`, and client secret must be specified. 
```
